packages/apps/tools/src/pages/home/index.tsx

```diff
@@ -6,11 +6,62 @@ import {
   helpCenterButtonClass,
   homeWrapperClass,
 } from '@/pages/home/styles.css';
-import { Breadcrumbs, Card, Heading, Table } from '@kadena/react-ui';
+import {
+  Accordion,
+  Box,
+  Breadcrumbs,
+  Card,
+  Grid,
+  Heading,
+  Table,
+} from '@kadena/react-ui';
 import useTranslation from 'next-translate/useTranslation';
+import Head from 'next/head';
+import Link from 'next/link';
+
 import type { FC } from 'react';
 import React, { useRef } from 'react';
 
+const faqs: Array<{ title: string; body: React.ReactNode }> = [
+  {
+    title: 'What can I do with the Faucet?',
```

Comment:
Will we add translation option for this?

Reply:
Added translations for these (by using [Trans](https://github.com/aralroca/next-translate#trans-component) component)